rules/falco_rules.yaml

```diff
@@ -1522,7 +1522,7 @@
   condition: (proc.aname[2]=rabbitmqctl and proc.cmdline startswith "sh -c ")
 
 - macro: run_by_appdynamics
-  condition: (proc.pname=java and proc.pcmdline startswith "java -jar -Dappdynamics")
+  condition: (proc.pexe endswith java and proc.pcmdline contains " -jar -Dappdynamics")
```

Comment:
This looks odd at first but at least we're guarded from processes that change their name and it doesn't introduce a false positive (I mean, unless we have something called `stuff-java` and has the same CLI options as these tools, which is very unlikely.